### PR TITLE
One-card tarot command & crimsoball cleanup

### PR DESCRIPTION
**Pull request by keli5**
- Clean up crimsoball command: `# list of answers (which I need to store somewhere besides in the function)` no more!
- A tarot command that uses a single card.
!! Tarot command as well as tarot reading for spread "one" need to be looked over, I don't know if present is the correct one to use, I don't know much about tarot cards

### CODE REVIEW

File: crimsobot/cogs/mystery.py [hunk 1]
```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
+        """ One card """  # this also needs to be improved
+
+        fp, descriptions = await tarot.reading(spread)
+        filename = 'reading.png'
```
Comment (kaylynn234) [inline crimsobot/cogs/mystery.py:37]: This seems wasteful.

Comment (keli5) [reply crimsobot/cogs/mystery.py:37]: what do you mean by wasteful?

File: crimsobot/cogs/mystery.py [hunk 2]
```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
```
Comment (kaylynn234) [inline crimsobot/cogs/mystery.py:33]: Is there a reason that `spread` is an argument? Being able to pass a different spread for `>tarot one` (e.g `>tarot one five`) seems odd.

File: crimsobot/cogs/mystery.py (hunk 1 above)
Comment (kaylynn234) [reply crimsobot/cogs/mystery.py:37]: Unneeded assignment - a string literal could be used in `f = discord.File(fp, filename)` (so that it becomes `f = discord.File(fp, 'reading.png')`) without issue.

File: crimsobot/cogs/mystery.py [hunk 3]
```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
...
+        f = discord.File(fp, filename)
+
+        embed = c.crimbed(
+            title="{}'s reading".format(ctx.author),
+            descr=None,
+            attachment=filename,
+            footer='Type ">tarot card" for more on a specific card.',
+        )
+
+        for card_tuple in descriptions:
```
Comment (kaylynn234) [inline crimsobot/cogs/mystery.py:47]: This seems wasteful again. I think I understand why this was done, but a loop when only one element is required seems silly.

File: crimsobot/cogs/mystery.py (hunk 1 above)
Comment (keli5) [reply crimsobot/cogs/mystery.py:37]: Filename is defined separately to provide this as the embed description as well.

File: crimsobot/cogs/mystery.py (hunk 2 above)
Comment (keli5) [reply crimsobot/cogs/mystery.py:33]: Other tarot subcommands do this as well, `spread` is used within the command itself to pass to `tarot.reading()`

File: crimsobot/cogs/mystery.py (hunk 3 above)
Comment (keli5) [reply crimsobot/cogs/mystery.py:47]: Fixed with 328d54e

File: crimsobot/utils/games.py [hunk 4]
```diff
@@ -13,6 +14,40 @@
...
+            'uhhhhhhhhh',
+            '<:uhhhh:495249068789071882>',
+            'eat glass!',
+            'it is important that you stop bothering me.',
+            'you CANNOT be serious',
+            'sure? how would i know?',
+            'what heck',
+            'random_response',  # leave this alone
+        ]
+    return random.choice(answer_list)
```
Comment (kaylynn234) [inline crimsobot/utils/games.py:48]: There should (potentially) be a newline before the return statement since that seems to be the formatting chosen the majority of the time for this project - though not all the time. Is the seeming inconsistency intentional @williammck?

Comment (keli5) [reply crimsobot/utils/games.py:49]: Fixed with a4c8767 as it's the majority, and generally looks better

File: crimsobot/cogs/mystery.py (hunk 2 above)
Comment (kaylynn234) [reply crimsobot/cogs/mystery.py:33]: That's odd behavior. I imagine it's a relic of the command before it was split into subcommands. No doubt @williammck  will see this but pinging anyway. Probably a good idea to think about cleaning this up, though I don't think it's worth opening an issue about.

Comment (keli5) [reply crimsobot/cogs/mystery.py:33]: Think I should quickly clean this up and commit it or leave that for later as it's a bit of a larger change?

Comment (kaylynn234) [reply crimsobot/cogs/mystery.py:33]: The diff looks ugly enough already, probably better suited for a later change but not sure. Also seems out-of-scope considering the current title for this PR.

Comment (keli5) [reply crimsobot/cogs/mystery.py:33]: Got it, will do this later

File: crimsobot/utils/games.py [hunk 5]
```diff
@@ -13,6 +14,41 @@
 DiscordUser = Union[discord.User, discord.Member]
 
 
+def return_cball_answer(ctx: commands.Context) -> str:  # function to give first answer a ctx to work with
+    # don't know if this is any better than just putting it
+    # inside of the crimsoball command
```
Comment (williammck) [inline crimsobot/utils/games.py:19]: no this is good, api abstractions like this are a welcome addition.

File: crimsobot/utils/tarot.py [hunk 6]
```diff
@@ -168,6 +168,16 @@ async def reading(spread: str) -> Tuple[Optional[io.BytesIO], List[Tuple[str, st
...
 
+    elif spread == 'one':
+        # a singular card on its own
+        # what is it? present?
+        bg_size = (w, h)
+        cards = await Deck.get_random_cards(1)
+        position = [
+            (0, 0)
+        ]
+        position_legend = ['PRESENT']  # just assuming once again
```
Comment (williammck) [inline crimsobot/utils/tarot.py:179]: https://discordapp.com/channels/552650672965943296/554799749593563137/727592213596340285
iudith said something about just keeping it empty, that's probably the best bet

File: crimsobot/utils/games.py [hunk 7]
```diff
@@ -13,6 +14,41 @@
 DiscordUser = Union[discord.User, discord.Member]
 
 
+def return_cball_answer(ctx: commands.Context) -> str:  # function to give first answer a ctx to work with
```
Comment (williammck) [inline crimsobot/utils/games.py:17]: nit: `return_cball_answer` is a bit of a weird method name, something like `get_crimsoball_answer` is probably better.

not a blocking suggestion though, this is fine.

File: crimsobot/cogs/mystery.py [hunk 8]
```diff
@@ -44,8 +65,8 @@ async def ppf(self, ctx: commands.Context, spread: str = 'ppf') -> None:
             footer='Type ">tarot card" for more on a specific card.',
         )
 
-        for card_tuple in descriptions:
-            embed.add_field(name=card_tuple[0], value='**{}**\n{}'.format(card_tuple[1], card_tuple[2]))
+        card_tuple = descriptions[0]
+        embed.add_field(name=card_tuple[0], value='**{}**\n{}'.format(card_tuple[1], card_tuple[2]))
```
Comment (williammck) [inline crimsobot/cogs/mystery.py:69]: looks like https://github.com/crimsobot/crimsoBOT/commit/328d54e0051d34878dfc897987c60f158e6f150c fixed the wrong method

File: crimsobot/cogs/mystery.py [hunk 9]
```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
```
Comment (williammck) [inline crimsobot/cogs/mystery.py:31]: `'A single reading'`, perhaps?

File: crimsobot/cogs/mystery.py [hunk 10]
```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
...
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
+        """ One card """  # this also needs to be improved
```
Comment (williammck) [inline crimsobot/cogs/mystery.py:34]: `"""This single-card reading is your answer to any question you may have."""` kinda fits

File: crimsobot/utils/tarot.py (hunk 6 above)
Comment (keli5) [reply crimsobot/utils/tarot.py]: done, https://github.com/crimsobot/crimsoBOT/pull/73/commits/42655b5ebd17245dad2fd437645405dfa4e2340f

File: crimsobot/utils/games.py (hunk 7 above)
Comment (keli5) [reply crimsobot/utils/games.py]: good idea; done in https://github.com/crimsobot/crimsoBOT/pull/73/commits/fb038dac1c73a155a20fdf5ca59e4f9fb582ac05

File: crimsobot/cogs/mystery.py (hunk 9 above)
Comment (keli5) [reply crimsobot/cogs/mystery.py]: sounds good, changed in https://github.com/crimsobot/crimsoBOT/pull/73/commits/12571c60232c4e2280f173bf90870845d1ac6791
